src/app/transactional-search/transactional-search.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit } from '@angular/core';
+import { TransactionSearchService } from '../transaction-search.service';
+import { ITransactionDetail } from './transaction.model';
+import { Response } from '@angular/http';
+import { HttpClientModule, HttpClient } from '@angular/common/http';
+
+@Component({
+  selector: 'app-transactional-search',
+  templateUrl: './transactional-search.component.html',
+  styleUrls: ['./transactional-search.component.css']
+})
+export class TransactionalSearchComponent implements OnInit {
+
+  searchText: any;
+  transactions: Array<ITransactionDetail>;
+  transactionsSecond: any[];
+  dummayArray = [];
```

Comment:
please give identical names to variables

Reply:
gave the identical name to the variables